pkg/ui/workspaces/cluster-ui/src/databaseDetailsPage/databaseDetailsPage.tsx

```diff
@@ -622,7 +622,10 @@ export class DatabaseDetailsPage extends React.Component<
             checkInfoAvailable(
               table.requestError,
               null,
-              table.details.nodesByRegionString || "None",
+              table.details.nodesByRegionString &&
+                table.details.nodesByRegionString.length > 0
```

Comment:
nit: you can consolidate this with `table.details.nodesByREgionString?.length`

Reply:
Done.